docs/tutorial/part-zero/index.md

```diff
@@ -1,235 +1,235 @@
+- [Installa Git su Linux](https://www.atlassian.com/git/tutorials/install-git#linux)
 
-## Using the Gatsby CLI
+## Utilizzo della Gatsby CLI
 
-The Gatsby CLI tool lets you quickly create new Gatsby-powered sites and run commands for developing Gatsby sites. It is a published npm package.
+Lo strumento Gatsby CLI ti permette in modo rapido di creare dei nuovi siti basati su Gatsby ed eseguire comandi per lo sviluppo dei siti Gatsby. È pubblicato come pacchetto npm.
 
-The Gatsby CLI is available via npm and should be installed globally by running `npm install -g gatsby-cli`.
+La Gatsby CLI è disponibile tramite npm e dovrebbe essere installata globalmente eseguendo `npm install -g gatsby-cli`.
 
-_**Note**: when you install Gatsby and run it for the first time, you'll see a short message notifying you about anonymous usage data that is being collected for Gatsby commands, you can read more about how that data is pulled out and used in the [telemetry doc](/docs/telemetry)._
+_**Nota**: quando installi Gatsby ed lo esegui per la prima volta, vedrai un breve messaggio che ti informa sulla raccolta di dati di utilizzo anonimi per i comandi Gatsby, puoi leggere di più su come i dati vengono collezionati e usati nel [documento telemetria](/docs/telemetry)._
 
-To see the commands available, run `gatsby --help`.
+Per vedere i comandi disponibili, esegui `gatsby --help`.
 
-![Check gatsby commands in terminal](05-gatsby-help.png)
+![Controlla nel terminale i comandi di gatsby](05-gatsby-help.png)
 
-> 💡 If you are unable to successfully run the Gatsby CLI due to a permissions issue, you may want to check out the [npm docs on fixing permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions), or [this guide](https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md).
+> 💡 Se non riesci ad eseguire Gatsby CLI a causa di un problema di permessi, prova a controllare il [documento su come sistemare i permessi di npm](https://docs.npmjs.com/getting-started/fixing-npm-permissions), o [questa guida](https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md).
 
-## Create a Gatsby site
+## Crea un sito Gatsby
 
-Now you are ready to use the Gatsby CLI tool to create your first Gatsby site. Using the tool, you can download “starters” (partially built sites with some default configuration) to help you get moving faster on creating a certain type of site. The “Hello World” starter you’ll be using here is a starter with the bare essentials needed for a Gatsby site.
+Ora sei pronto per usare la Gastby CLI per creare il tuo primo sito Gatsby. Usando questo strumento, puoi scaricare gli “starters” (siti costrutiti in modo parziale con una configruazione di base) per aiutarti a creare determinati tipologie di sito in maniera più veloce. Lo starter “Hello World” che utilizzerai qui è uno starter con gli elementi essenziali necessari per un sito Gatsby.
```

Comment:
Le parole straniere non vanno declinate in italiano: il plurale è sempre "starter"